dev-python/flask-paginate/flask-paginate-0.6.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( pypy3 python3_{6,7} )
+
+inherit distutils-r1
+
+MY_COMMIT="510ad833106134711868653fb597bf75ea8ac34f"
```

Comment:
This commit is from 0.5.3 version (https://github.com/lixxu/flask-paginate/commit/510ad833106134711868653fb597bf75ea8ac34f)